	- set HOSTCC to the right value for OpenADK
	- maradns, y u no use HOSTCC when appropriate?!
--- maradns-2.0.09.orig/deadwood-3.2.05/src/Makefile	2011-12-21 20:44:37.000000000 +0100
+++ maradns-2.0.09/deadwood-3.2.05/src/Makefile	2014-06-19 02:48:35.375046777 +0200
@@ -5,7 +5,7 @@
 # compiling.  When cross-compiling, we still need to compile and
 # run programs on the build machine, so change the next line when
 # cross-compiling.
-HOSTCC=$(CC)
+HOSTCC=$(CC_FOR_BUILD)
 
 OBJS=DwStr.o \
 	DwMararc.o \
@@ -68,7 +68,7 @@ DwSys.o:	DwSys.c DwStr.h
 	$(CC) $(FLAGS) -Wall -c -o DwSys.o DwSys.c
 
 RandomPrime:	RandomPrime.c
-	$(CC) -O3 -o RandomPrime RandomPrime.c
+	$(HOSTCC) -O3 -o RandomPrime RandomPrime.c
 
 DwRandPrime.h: RandomPrime
 	if [ -e /dev/urandom ] ; then ./RandomPrime > DwRandPrime.h ; fi
